tests/test_util.py

```diff
@@ -108,11 +108,27 @@ def test_message_normalization():
     The truncate() method of io.BufferedReader() should raise
     UnsupportedOperation when it is called on a read-only
     io.BufferedReader() instance.
-    
+
     Automerge-Triggered-By: @methane"""
     )
     assert util.normalize_message(message) == expected_message
 
+    message = textwrap.dedent(
+        """
+    Some commit message.
+
+
+    <!-- gh-issue-number: gh-93516 -->
+    * gh-93516
```

Comment:
Should this be 
```suggestion
    * Issue: gh-93516
```

? See also PR https://github.com/python/bedevere/pull/480